samples/openapi3/client/petstore/python-experimental/tests_manual/test_fake_api.py

```diff
@@ -400,6 +400,38 @@ def close(self):
         api_response.body.close()
         os.unlink(api_response.body.name)
 
+        file1 = open(file_path1, "rb")
+        streamable_body = StreamableBody(file1)
+        expected_filename = "the_file.ext"
+
+        no_filename_mock_response = self.response(
+            streamable_body,
+            content_type='application/octet-stream',
+            headers={'content-disposition': f'attachment'},
```

Comment:
nit: one does not need a f-string for the content disposition value. How about removing the f prefix from that string?